arm-network/2016-12-01/swagger/virtualNetworkGateway.json

```diff
@@ -1600,6 +1611,129 @@
       ],	  
       "description": "Response for GetConnectionSharedKey API service call"
     },
+    "IpsecPolicy": {
+      "properties": {
+        "SALifeTimeSeconds": {
```

Comment:
All the properties are in PascalCase. On the wire everything should be in camelCase as per the convention. JSON object is like a case sensitive dictionary.
"SALifeTimeSeconds" --> **"s** ALifeTimeSeconds"